### PR TITLE
[IMP] project: back2basics

### PR DESCRIPTION
**Pull request by arj-odoo**
taskid: 2299401

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by tivisse** [comment]
@robodoo r+

File: addons/pad_project/models/project.py [hunk 1]
```diff
@@ -42,8 +42,11 @@ def _get_pad_content(self):
 
 
 class ProjectProject(models.Model):
-    _inherit = "project.project"
+    _name = "project.project"
+    _inherit = ["project.project", 'pad.common']
+    _description = 'Project'
 
+    description_pad = fields.Char('Pad URL', pad_content_field='description', copy=False)
```
Comment (tivisse) [inline addons/pad_project/models/project.py:49]: util.create_column.

File: addons/project/models/project.py [hunk 2]
```diff
@@ -155,11 +155,17 @@ def _get_default_favorite_user_ids(self):
         return [(6, 0, [self.env.uid])]
 
     name = fields.Char("Name", index=True, required=True, tracking=True)
-    description = fields.Text()
+    description = fields.Html()
```
Comment (tivisse) [inline addons/project/models/project.py:158]: We should handle the type conversion ? 

Something like
```
        cr.execute("""
            UPDATE project_issue
               SET description = {}
             WHERE description IS NOT NULL
        """.format(util.pg_text2html('description')))
```

File: addons/project/models/project.py [hunk 3]
```diff
@@ -155,11 +155,17 @@ def _get_default_favorite_user_ids(self):
...
     active = fields.Boolean(default=True,
         help="If the active field is set to False, it will allow you to hide the project without removing it.")
     sequence = fields.Integer(default=10, help="Gives the sequence order when displaying a list of Projects.")
     partner_id = fields.Many2one('res.partner', string='Customer', auto_join=True, tracking=True, domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]")
+    partner_email = fields.Char(
+        compute='_compute_partner_email', inverse='_inverse_partner_email',
+        string='Email', readonly=False, store=True)
```
Comment (tivisse) [inline addons/project/models/project.py:165]: util.create_column + compute the value in SQL

File: addons/project/models/project.py [hunk 4]
```diff
@@ -155,11 +155,17 @@ def _get_default_favorite_user_ids(self):
...
+    partner_phone = fields.Char(
+        compute='_compute_partner_phone', inverse='_inverse_partner_phone',
+        string="Phone", readonly=False, store=True)
```
Comment (tivisse) [inline addons/project/models/project.py:168]: util.create_column + compute the value in SQL

File: addons/project/models/project.py (hunk 2 above)
Comment (tivisse) [reply addons/project/models/project.py:158]: Confirmed by chs.

**Comment by Xavier-Do** [comment]
@tivisse The merge method is not set here in case you forgot :)

**Comment by tivisse** [comment]
@robodoo rebase-ff

**Pull request closed without merging**